libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -61,6 +61,21 @@ core::ProcessGroup *YamlConfiguration::parseRootProcessGroupYaml(YAML::Node root
 
   this->name_ = flowName;
 
+  if (rootFlowNode["onschedule retry interval"]) {
```

Comment:
I guess the logic is that this is something that is optional? This makes a lot of sense if requested via the flow configuration but it does not always make sense in the guise of distributed containers/systems.

Reply:
Something configurable. You are free to set 0, which is going to turn it off. 
Although to match NiFi behaviour retry should be the default way to do things. In case a processor can't connect (for eg. the destination is temporary down), we shouldn't give up entirely.